.yarnclean

```diff
@@ -0,0 +1,42 @@
+# test directories
```

Comment:
@Peter-holyoke 
Was this file generated by `yarn init`?  I'm not quite sure what it is for?